src/ext/oblivc/obliv_bits.c

```diff
@@ -886,11 +886,13 @@ void splitYaoProtocolExtra(ProtocolDesc* pdout, ProtocolDesc * pdin) {
   ypdout->extra = NULL;
   ypdout->fixedKeyCipher = ypdin->fixedKeyCipher;
   if (pdout->thisParty == 1) {
-    gcry_randomize(ypdout->R,YAO_KEY_BYTES,GCRY_STRONG_RANDOM);
+    gcry_randomize(&ypdout->gcount,sizeof(ypdout->gcount),GCRY_STRONG_RANDOM);
```

Comment:
Ah I see it's fixed here. Maybe squash the two commits before merge, so we won't have it in a broken stage in-between.

And ... if it's still untested, do test it. Feel free to add a simple test to test/oblivc.